doc/src/devdocs/stdio.md

```diff
@@ -2,37 +2,37 @@
 
 ## Libuv wrappers for stdio
 
-`julia.h` defines [libuv](http://docs.libuv.org) wrappers for the `stdio.h` streams:
+julia.h defines [libuv](http://docs.libuv.org) wrappers for the stdio.h streams:
```

Comment:
> removing the code formatting from file names and paths that are not code. It provides a better visual distinction IMO, and that's more common in other parts of the docs

I'm still not seeing this. Grepping for `\.jl` or `\.c` under docs, far more of the hits that aren't part of a link are code highlighted than not.